### PR TITLE
allow Symfony 7

### PR DESCRIPTION
**Pull request by tacman**
It appears that tests pass and the bundle installs.

I'm working on a small Symfony 7 demo but running into unrelated issues.  But I figured I'd get the ball rolling with these small changes.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "name": "knplabs/knp-menu-bundle",
+    "minimum-stability": "beta",
```
Comment (Chris53897) [inline composer.json:3]: Do we need this line? Or could this be added to the CI only?

Comment (tacman) [reply composer.json:3]: Probably not, I had that (and the symfony extra) to make sure that tests pass.  How is that added to CI?

Comment (Chris53897) [reply composer.json]: One way could be like this. https://github.com/tattali/CalendarBundle/pull/64/files#diff-f47fc96cfa328e123eaec41fe0ef2d23567991431dbdeef87da6c024dd103c08R61

Or a seperate CI-File for dev

**Review by garak — APPROVED** [review]

**Comment by derrabus** [comment]
The new CI job fails.

**Comment by tacman** [comment]
Hmm, maybe this line needs to change:

Run sed -ri 's/"symfony\/(expression-language|framework-bundle|templating)": "(.+)"/"symfony\/\1": "'7'"/' composer.json;

is it supposed to be ^7.0?  I'll try that, and I'll try to figure out how to trigger the action from my fork.

**Comment by tacman** [comment]
Thanks, passing now.

**Comment by tacman** [comment]
Fixes #467

**Review by derrabus — APPROVED** [review]

**Comment by trsteel88** [comment]
@garak are you able to release a new version for Symfony 7?

**Pull request merged**